nixos/modules/services/hardware/sane.nix

```diff
@@ -21,7 +21,11 @@ in
     hardware.sane.enable = mkOption {
       type = types.bool;
       default = false;
-      description = "Enable support for SANE scanners.";
+      description = ''
+        Enable support for SANE scanners.
+
+        Users in the "scanner" group will gain access to the scanner.
```

Comment:
Note that paragraph breaks like this won't show up in the generated docs. I think adding `<para>` does work.